scripts/scil_remove_outliers_ransac.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Remove outliers from image using the RANSAC algorithm.
+The RANSAC algorithm parameters are sensitive to the input data.
+
+NOTE: Current default parameters are tuned for ad/md/rd images only.
```

Comment:
Maybe a warning if the input image average signal (excluding 0) are above 0.01.

I cannot really make it work for a FA, so I would write that it is not recommended to use on something else.